### DISCUSSION TITLE
Welcome to Amplify GenAI Discussions!

### DISCUSSION THREAD

**Discussion by karnsab** [gaiin-platform/.github#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krishagel** [comment]
Welcome everyone!

I'm Kris Hagel, the Chief Information Officer for the Peninsula School District in Gig Harbor, WA. We have been testing Amplify since last spring and about six weeks ago we rolled it out into production for all staff in our K-12 School District. We have two big upcoming staff trainings in early December where we will be digging into Amplify with all administrators across the district and building out assistants to help staff in our district. We have an extensive list of ideas and are just getting started building them out. I have a lot of ideas for extending Amplify, but need to spend more time understanding the code to be able to do that.

**Reply by ejones40** [reply]
Hey Kris - glad to hear it! Feel free to add any of the ideas that you have to the issues boards on the repos (front-end, back-end, iac) and they will be tagged as `enhancement`.

**Comment by UTCbridget** [comment]
Hello, I'm a frontend developer unfamiliar with React, but familiar with Tailwindcss/npm. I'm trying to brand our university's Amplify application, but my local changes in the home.tsx are not showing. I'm sure I'm missing a process. Is there documentation as to how to brand Amplify?

**Reply by ejones40** [reply]
Hey Bridget - I spoke w/ a member of the UTC team recently, and it sounds like you guys were able to figure this out. Would you be open to submitting a PR to `dev` so that others can also custom-brand the application?

**Reply by UTCbridget** [reply]
@ejones40, I'm still working on this for UTC, however, my rebranding mostly is changing up the tailwind code. The issue with the home.tsx changes not showing was something different in that I had two containers in docker and was displaying the wrong container. Novice mistake. :\

**Reply by ejones40** [reply]
Got it - thanks for sharing! Branding configuration is something that I know is a desired feature for folks - glad you were able to figure it out.
